README.md

```diff
@@ -8,29 +8,22 @@ A booking system for the yearly event **Allaktivitetsdag** at Tullinge gymnasium
 
 ## Requirements
 
-- Docker
+- Docker & docker-compose
 - Python 3
 - Python libraries (look in `requirements.txt`)
-- MySQL database (container can be started with docker-compose)
-- Redis instance (container can be started with docker-compose)
 
-## Database Configuration
-
-This is how the db_config.py file is structured.
-
-```python
-DB_Server = '[hostname]'
-DB_Name = '[name of database]'
-DB_Username = '[username with access to database]'
-DB_Password = '[password for user]'
-```
-
-## Instructions
-
-1. `docker-compose up` starts the database container
-2. `python scripts/setup_db.py` creates the database structure
-3. `python scripts/insert_demo_data.py` seeds the database with demo data
-4. `python app.py` starts the web app
+### Instructions (running locally)
+
+1. `docker-compose up -d`
+2. `python scripts/setup_db.py`
+3. `python scripts/seed_db.py`
+4. `python app.py`
+   
+### Instructions (deployment)
```

Comment:
I believe there should be a space here, between the header and the content (see [markdownlint](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines)). This is really minor :smile: